.github/workflows/build.yml

```diff
@@ -21,7 +21,8 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        compiler: [AC6, GCC, CLANG]
+        compiler: [AC6]
```

Comment:
Can we have an alternative vcpkg-configuration.json yml file when running the CI system, where all three compilers are loaded and tested?